node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/DockerHostCapacity.java

```diff
@@ -123,28 +123,24 @@ int freeCapacityInFlavorEquivalence(ResourceCapacity freeCapacity, Flavor flavor
 
     /**
      * Calculate the remaining capacity for the dockerHost.
-     * @param dockerHost The host to find free capacity of.
      *
+     * @param dockerHost The host to find free capacity of.
      * @return A default (empty) capacity if not a docker host, otherwise the free/unallocated/rest capacity
      */
-    public ResourceCapacity freeCapacityOf(Node dockerHost, boolean treatInactiveOrRetiredAsUnusedCapacity) {
+    public NodeResources freeCapacityOf(Node dockerHost, boolean includeInactive) {
         // Only hosts have free capacity
-        if (!dockerHost.type().equals(NodeType.host)) return ResourceCapacity.NONE;
+        if ( ! dockerHost.type().equals(NodeType.host)) return new NodeResources(0, 0, 0);
```

Comment:
It doesn't make sense any more because empty isn't well defined for disk-speed. Actually two of the three uses of this should set disk-speed, and I did that now.